Examples/NavigationTransitionsExample/Transition/Transition/InteractiveZoomTransitionController.swift

```diff
@@ -66,12 +66,23 @@ extension InteractiveZoomTransitionController : SharedElementProvider {
         
         let initialFrame: CGRect
         let targetFrame: CGRect
+
+        // Get top margin, depending on possible safe area
+        var yMargin: CGFloat
+
+        if #available(iOS 11.0, *) {
+            yMargin = toViewController.collectionView.safeAreaInsets.top
+        } else {
+            yMargin = 0.0
+        }
+        
+        yMargin += CollectionViewController.margin
```

Comment:
@mvolpato More succinct would be to initialize `yMargin` with `CollectionViewController.margin` and only add `safeAreaInsets.top` on iOS 11+.